src/util/profile/prof_file.c

```diff
@@ -60,6 +59,10 @@ int profile_library_initializer(void)
 
     return k5_mutex_finish_init(&g_shared_trees_mutex);
 }
+
+#ifdef _WIN32
```

Comment:
I don't understand how this code is different from other uses of MAKE_FINI_FUNCTION such that there is a warning here but not elsewhere.  (For example, crypto_libinit.c.)

Unconditionally using __attribute__(unused) breaks on non-gcc/clang compilers, as witnessed by the appveyor build failure.